netbox/netbox/views/generic/bulk_views.py

```diff
@@ -511,25 +526,11 @@ def _update_objects(self, form, request):
                 obj.snapshot()
 
             # Update standard fields. If a field is listed in _nullify, delete its value.
-            for name in standard_fields:
-
-                try:
-                    model_field = self.queryset.model._meta.get_field(name)
-                except FieldDoesNotExist:
-                    # This form field is used to modify a field rather than set its value directly
-                    model_field = None
-
+            for name, model_field in model_fields.items():
                 # Handle nullification
                 if name in form.nullable_fields and name in nullified_fields:
-                    if isinstance(model_field, ManyToManyField):
-                        getattr(obj, name).set([])
-                    else:
+                    if not isinstance(model_field, ManyToManyField):
```

Comment:
Might be able to just dump this as model_fields shouldn't contain m2m fields anymore.